src/runtime/virtcontainers/hypervisor.go

```diff
@@ -450,6 +450,11 @@ type HypervisorConfig struct {
 	// GuestMemoryDumpPaging is used to indicate if enable paging
 	// for QEMU dump-guest-memory command
 	GuestMemoryDumpPaging bool
+
+	// Enable guest protection.
+	// Enable or disable different hardware features, ranging
+	// from memory encryption to both memory and CPU-state encryption and integrity.
+	GuestProtection bool
```

Comment:
Same comment as in configuration file. I'd rather make room for more than `on/off` here.

Reply:
what do you mean? are you expecting this to be a string instead of a boolean ?

Reply:
cc @c3d

Reply:
and here